samples/snippets/upload_model_explain_image_managed_container_sample_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from uuid import uuid4
+
+import pytest
+
+import delete_model_sample
```

Comment:
remove this import

Reply:
Which import? Both are used. @pytest.fixture requires pytest for example and existing tests have this import?

Reply:
the `delete_model_sample` import.  it should not be used.  please compare with `upload_model_sample_test.py` in the main branch of the repo: https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/upload_model_sample_test.py

Reply:
I would've thought the upload_model_sample_test you linked would thru an exception at the end -- because it references an unintialized variable shared_state:

shared_state["model_name"] = helpers.get_name(out, key="model")